.env.production

```diff
@@ -15,3 +15,11 @@ REACT_APP_DOMAIN_PREFIX_PROD=cow.trade
 REACT_APP_GOOGLE_ANALYTICS_ID_DEV="UA-190948266-3"
 REACT_APP_GOOGLE_ANALYTICS_ID_STAGING="UA-190948266-4"
 REACT_APP_GOOGLE_ANALYTICS_ID_PROD="UA-190948266-1"
+REACT_APP_PORTIS_ID="c0e2bf01-4b08-4fd5-ac7b-8e26b58cd236"
+REACT_APP_FORTMATIC_KEY="pk_live_F937DF033A1666BF"
+REACT_APP_GOOGLE_ANALYTICS_ID="UA-190948266-1"
+
+# API
+#REACT_APP_API_BASE_URL_MAINNET=https://protocol-mainnet.dev.gnosisdev.com/api/v1
+#REACT_APP_API_BASE_URL_RINKEBY=https://protocol-rinkeby.dev.gnosisdev.com/api/v1
+#REACT_APP_API_BASE_URL_XDAI=https://protocol-xdai.dev.gnosisdev.com/api/v1
```

Comment:
Are you going to leave these commented out?

Reply:
yep, is just an example